browsers/duckduckgo.safariextension/js/content-script.js

```diff
@@ -35,22 +29,36 @@ var onBeforeLoad = (e) => {
         if (block.cancel) {
             console.log(`DDG BLOCKING ${e.url}`)
             e.preventDefault()
+        } else if (block.redirectUrl) {
+            console.log(`DDG BLOCKING AND USING SURROGATE ${e.url}`)
+            setTimeout(redirectSrc.bind(undefined, e.target, block.redirectUrl), 1)
```

Comment:
Is the request still going through here?

Reply:
oof, good point, I should have checked that.

Reply:
might need to add the e.preventDefault(), checking now